dask/array/tests/test_reductions.py

```diff
@@ -422,3 +422,10 @@ def test_array_cumreduction_out(func):
     x = da.ones((10, 10), chunks=(4, 4))
     func(x, axis=0, out=x)
     assert_eq(x, func(np.ones((10, 10)), axis=0))
+
+
+def test_no_partials():
+    cloudpickle = pytest.importorskip('cloudpickle')
```

Comment:
Array graphs should be completely pickle serializable (unless things changed since I went through and poked at this). May be fine to use `pickle` instead of `cloudpickle`.